src/Authentication/App/Controllers/LoginController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Authentication\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Authentication\App\Requests\LoginRequest;
+use Lightit\Authentication\App\Rescources\LoginResource;
+use Lightit\Authentication\Domain\Actions\LoginAction;
+
+class LoginController
+{
+    public function __invoke(LoginRequest $request, LoginAction $loginAction): JsonResponse
+    {
+        $dto = $request->toDto();
+
+        $loginDto = $loginAction->execute($dto);
```

Comment:
```suggestion
        $credentials = $request->toDto();
        $loginResult = $loginAction->execute($credentials);
```
Consider renaming those variables for better readability, the above is just an example